source/extensions/transport_sockets/tls/cert_validator/spiffe/spiffe_validator.cc

```diff
@@ -279,18 +425,22 @@ std::string SPIFFEValidator::extractTrustDomain(const std::string& san) {
   }
 
   auto pos = san.find('/', prefix.size());
-  if (pos != std::string::npos) {
+  if (pos == std::string::npos) {
+    // If no additional slash is found, return everything after the prefix
+    return san.substr(prefix.size());
+  } else {
```

Comment:
Seems like this is an unnecessary change and bring a behavior change?

Reply:
In my testing I found this behavior of not treating with and without trailing slash as equivalent. I believe it's a bug, but maybe a SPIFFE expert would disagree? I could remove it from this and propose it in a separate PR if that's better.

Reply:
> In my testing I found this behavior of not treating with and without trailing slash as equivalent. I believe it's a bug, but maybe a SPIFFE expert would disagree? I could remove it from this and propose it in a separate PR if that's better.

Yeah, I think make it an independent PR would be better. Thanks.